### DISCUSSION TITLE
blank workflow after updating to 2.8

### DISCUSSION THREAD

**Discussion by ablot** [bonsai-rx/bonsai#1538] [Q&A] [answered]
Hi all, 
we installed bonsai 2.8 recently and some of our workflow don't seem to load. They open with a totally blank workflow editor, but no error in the command line. If we saveas, the editor generates a empty .bonsai file.
The files used to work in v2.7 and opening them with text editor, I can confirm that they are not empty. Do you have any idea on how to debug that?
I've attached one example workflow. The one crashing seem to have some bonvision but not sure if it is the issue

[AcquireEphys_VR _C#_testing.zip](https://github.com/bonsai-rx/bonsai/files/12430397/AcquireEphys_VR._C._testing.zip)

**Comment by bruno-f-cruz** [comment, marked as answer]
Not exactly sure what is going on. Meanwhile, a valid workaround seems to be to copy the XML text directly to a Bonsai canvas and re-save the file.

**Reply by ablot** [reply]
That does work! ✨ magic ✨

**Reply by glopesdev** [reply]
It seems like opening the workflow from the File > Open menu works on the original file so it's not like the XML is really invalid. Other workflow files I have on my computer don't have this behavior, so it seems to be something about this specific file that is triggering the IDE. This is indeed strange, I will investigate further and possibly raise an issue.

**Reply by glopesdev** [reply]
@ablot @bruno-f-cruz FYI I have pinned down this issue to the way we construct the command-line arguments in the new child-process bootstrapper. I have opened https://github.com/bonsai-rx/bonsai/issues/1541 to track this for our first-patch release to come out soon.

**Comment by mohebial** [comment]
I have a similar issue. Workflows created with 2.7 open as blank. When I open them through the Open menu though, I can open them. And when I try to call them via CLI, Bonsai opens as blank again... Any thoughts? Thanks!

**Comment by trose-neuro** [comment]
this seems related: https://github.com/bonsai-rx/bonsai/issues/1539#issue-1869413998. Also: We currently have frequent crashes during loading of workflows. When they load, everything runs well.
